framework/Samplers/LimitSurfaceSearch.py

```diff
@@ -486,20 +490,15 @@ def localStillReady(self,ready): #,lastOutput=None
             self.invPointPersistence[gridID] = (maxPers-self.invPointPersistence[gridID])/maxPers
         else:
           self.firstSurface = False
-        if self.solutionExport!=None:
-          for varName in self.solutionExport.getParaKeys('inputs'):
-            for varIndex in range(len(self.axisName)):
-              if varName == realAxisNames[varIndex]:
-                if cnt == 0:
-                  self.solutionExport.removeInputValue(varName)
-                for value in self.surfPoint[gridID][:,varIndex]:
-                  self.solutionExport.updateInputValue(varName,copy.copy(value))
-          # to be fixed
-          if cnt == 0:
-            self.solutionExport.removeOutputValue(self.goalFunction.name)
-          for index in range(len(evaluations[gridID])):
-            self.solutionExport.updateOutputValue(self.goalFunction.name,copy.copy(evaluations[gridID][index]))
-        cnt+=1
+        if self.solutionExport is not None:
+          # construct the realizations dict
+          localRlz = {varName: (self.surfPoint[gridID][:,varIndex] if varName not in rlz else np.concatenate(( rlz[varName],self.surfPoint[gridID][:,varIndex] )) ) for varIndex,varName in enumerate(realAxisNames) }
```

Comment:
Code standard: "Lines longer than 120 characters should be split if possible."  This line is over 200 characters.